source/termination_criteria/steady_heat_flux.cc

```diff
@@ -0,0 +1,212 @@
+/*
+  Copyright (C) 2019 - 2020 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#include <aspect/termination_criteria/steady_heat_flux.h>
+#include <aspect/postprocess/heat_flux_map.h>
+
+namespace aspect
+{
+  namespace TerminationCriteria
+  {
+    namespace internal
+    {
+      void trim_time_heat_flux_list (const double necessary_time_in_steady_state,
+                                     std::list<std::pair<double, double> > &time_heat_flux_list)
+      {
+        // Remove old times until we're at the correct time period
+        // but ensure at least two entries remain in the list (one old, one current timestep)
+        auto it = time_heat_flux_list.begin();
```

Comment:
Out of curiosity, does `it` refer to "iterator time"?

Reply:
I think it just stands for iterator, but I only copied this from the steady state temperature plugin.